src/bun.js/test/jest.zig

```diff
@@ -1539,6 +1540,15 @@ pub const TestRunnerTask = struct {
             .skip => Jest.runner.?.reportSkip(test_id, this.source_file_path, test_.label, describe),
             .todo => Jest.runner.?.reportTodo(test_id, this.source_file_path, test_.label, describe),
             .fail_because_todo_passed => |count| {
+                var scopes_stack = std.BoundedArray(*DescribeScope, 64).init(0) catch unreachable;
+                var parent_: ?*DescribeScope = describe;
+                while (parent_) |scope| {
+                    scopes_stack.append(scope) catch break;
+                    parent_ = scope.parent;
+                }
+                const scopes: []*DescribeScope = scopes_stack.slice();
+                for (1..scopes.len) |_|
+                    Output.prettyError("  ", .{});
```

Comment:
I think you can remove the `BoundedArray` and just go up through the parents, skipping the first, something like:
```suggestion
                var parent_: ?*DescribeScope = describe;
                while (parent_) |scope| {
                    if (describe != scope) {
                      Output.prettyError("  ", .{});
                    }
                    parent_ = scope.parent;
                }
```